locales/fr-FR/messages.po

```diff
@@ -120,7 +120,7 @@ msgid "Item(s)"
 msgstr "Item(s)"
 
 msgid "Items"
-msgstr "Items"
+msgstr "Articles"
```

Comment:
`messages_po.js` has the same identifier.
I'm pretty sure it should be updated as well.

Reply:
Yes, you must run `php tao/scripts/taoTranslate.php -a=compile -e=taoCe -l=fr-FR` in order to get the `message_po.js` file updated.

More info here: https://hub.taotesting.com/articles/documentation-for-core-components/internationalization-i18n#how-to-update-or-add-translations

Reply:
message_po.js generated and I update for all languages